modules/e2ee/E2EEncryption.js

```diff
@@ -88,6 +88,8 @@ export class E2EEncryption {
         this._olmAdapter.on(
             OlmAdapter.events.PARTICIPANT_KEY_UPDATED,
             this._onParticipantKeyUpdated.bind(this));
+
+        this._onParticipantE2EEPropertyChanged.bind(this);
```

Comment:
Let's inline this function since it's very small and used in a single place please.